src/font/SharedGridSet.zig

```diff
@@ -324,7 +327,7 @@ fn collection(
             _ = try c.add(
                 self.alloc,
                 .regular,
-                .{ .fallback_deferred = face },
+                .init(.{ .fallback_deferred = face }),
```

Comment:
We probably want to opt out of scaling for the emoji font too? (Not gonna push the change right now as I know you're in the middle of reviewing.)

```suggestion
                .initWithScaleReference(.{ .fallback_deferred = face }, .em_size),
```